functions/package.json

```diff
@@ -1,33 +1,45 @@
 {
   "name": "functions",
+  "main": "lib/index.js",
   "description": "Cloud Functions for Firebase",
   "scripts": {
     "serve": "firebase serve --only functions",
     "shell": "firebase experimental:functions:shell",
     "emulators": "firebase emulators:start --config ../firebase.local.json --project fake-project-id",
     "deploy": "firebase deploy --only functions",
     "logs": "firebase functions:log",
-    "test": "mocha --reporter list"
+    "test": "mocha --reporter list",
+    "build": "npm run lint && tsc"
```

Comment:
`npm run build` fails since `lint` isn't defined here as a script.

Reply:
Got the eslint working. I don't know how to set up `gts` and I'm really bad at this. Let's stick to this for now, shall we?

Reply:
Sgtm, we just need to add `eslint` as a dependency for the build to pass; build fails for me, but I think it may you may have it install globally in your env.